source/analytics/reports/jurisdictions/montana/v182 Midis Jurisdiction/sql/SQLQuery_Completeness_of_Data.sql

```diff
@@ -0,0 +1,70 @@
+DECLARE @date_value date = {{Enter_Date}};
+DECLARE @jurisdiction_value NVARCHAR(MAX) = {{Jurisdiction}};
+
+--Jurisdiction 
+SELECT
+    I.JURISDICTION_NM AS Jurisdiction,
+    COUNT(*) AS '#Cases',
+    -- (COUNT(CASE WHEN INVCTE.CC_CLOSED_DT IS NOT NULL THEN 1 END) * 100.0)/COUNT(*) AS '% LTF',
```

Comment:
If these are tentative information and we need to have revisions, then consider adding a `ToDo` section  wherever applicable.